Services/Notifications/classes/class.ilObjNotificationAdminGUI.php

```diff
@@ -1,260 +1,147 @@
 <?php
```

Comment:
Please add `strict_types` (for every file in `Services/Notifications`).

Reply:
Same here. I would outsource this into a major refactoring/PHP8 PR